widget/table.go

```diff
@@ -41,24 +44,43 @@ func NewTable(length func() (int, int), create func() fyne.CanvasObject, update
 	return t
 }
 
+// ClearSelection will clear any cell selection.
+func (t *Table) ClearSelection() {
+	t.SelectedRow = -1
+	t.SelectedColumn = -1
+
+	if t.OnCellSelected != nil {
+		t.OnCellSelected(-1, -1)
```

Comment:
I'm not sure we should be calling OnCellSelected when we are quite literally doing the opposite of what the api suggests by unselecting?

Reply:
I think that in conversation we agreed that this was consistent, so can you approve this please?